src/objects/obstacles/pipes.ts

```diff
@@ -1,25 +1,89 @@
 import { BaseObject } from "../../utils/interfaces/object-abstract-class.ts";
 import * as CONFIG from '../../utils/configuration.ts'
+import { Game } from "../../scenes/Game.ts";
+import { generateRandom } from "../../utils/generateRandom.ts";
 
 export class Pipes extends BaseObject {
-    pipe: Phaser.Physics.Arcade.Sprite;
+
+    private pipes: Phaser.Physics.Arcade.Sprite[][] = [];
     
     constructor(scene: Phaser.Scene) {
         super(scene);
         this.init();
     }
 
     init() {
-        this.pipe = this.scene.physics.add.sprite(
-            this.scene.cameras.main.width / 2, 
-            this.scene.cameras.main.height, 
+    }
+
+    private createPipe (x: number, y: number, rotation: number ): Phaser.Physics.Arcade.Sprite {
+        const pipe = this.scene.physics.add.sprite(
+            x, 
+            y, 
             'pipe'
         )
-        .setDepth(CONFIG.PIPE_DEPTH);
+        .setDepth(CONFIG.PIPE_DEPTH)
```

Comment:
generally (but not required) you want to indent lines that continue the previous statement for easy reading
```suggestion
         .setDepth(CONFIG.PIPE_DEPTH)
          .setRotation(rotation);
``